orbit/pkg/bitlocker/bitlocker_management_windows.go

```diff
@@ -0,0 +1,574 @@
+//go:build windows
+// +build windows
```

Comment:
You can remove this `+build` version, this was for support with old Go versions but now that we use Go 1.20+ the `go:build` comment is enough.